org.adempiere.base/src/org/adempiere/base/DefaultModelFactory.java

```diff
@@ -322,4 +322,14 @@ public PO getPO(String tableName, ResultSet rs, String trxName) {
 			s_log.log(Level.SEVERE, "(rs) - Not found - Table=" + tableName);
 		return null;
 	}
+
+	@Override
+	public PO getPO(String tableName, String uuID, String trxName)
```

Comment:
this should be moved to the interface as a default method. very unlikely that any model factory implementation will need to override this.